src/test/regress/expected/isolation_add_node_vs_reference_table_operations.out

```diff
@@ -5,7 +5,7 @@ create_reference_table
 
                
 step s2-load-metadata-cache: 
-	COPY test_reference_table FROM PROGRAM 'echo "1\n2\n3\n4\n5"';
+	COPY test_reference_table FROM PROGRAM 'seq 1 5';
```

Comment:
```seq``` does not work in windows. We would need to it from ```MinGW```.  Perhaps we can create a script in regression test like ```tmp_check/tmp-bin/psql``` to get this behavior.